ui/api/src/main/java/org/jboss/forge/addon/ui/input/SelectComponent.java

```diff
@@ -30,4 +31,9 @@
    Converter<VALUETYPE, String> getItemLabelConverter();
 
    IMPLTYPE setItemLabelConverter(Converter<VALUETYPE, String> converter);
+
+   Converter<VALUETYPE, String> getInteractionItemLabelConverter();
+
```

Comment:
I may be wrong, but I find less likely that the addon developer will worry about setting a converter for the Interaction presentation. Maybe we should change the UIContext.getProvider().isGUI() to return true when interaction mode is enabled (just for the beautifulness of option values) ? WDYT @lincolnthree ?